packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -420,13 +446,17 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
             <SectionFooter className={classes.organizerActions}>
               {canCreateEvent &&
                 (!isEAForum || isAdmin || isGroupAdmin) && <SectionButton>
-                  <Link to={`/newPost?${qs.stringify({eventForm: true, groupId})}`}>
-                    New event
-                  </Link>
+                  <HoverOver title={<div>
+                    Note: If this is a recurring event, you might want to open the menu on a previous event and choose Duplicate Event.
+                  </div>}>
```

Comment:
Note that this is currently not forum-gated and it looks pretty bad on EAF, so I suggest forum-gating it

<img width="697" alt="Screenshot 2024-10-27 at 1 36 03 AM" src="https://github.com/user-attachments/assets/5749705a-cbc5-461a-9704-e767fff73c32">